### PR TITLE
Ldanzinger/query sublayer

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    // connect to city sublayer query signal
+    connect(m_citiesTable, &ServiceFeatureTable::queryFeaturesCompleted, this, [this](QUuid, FeatureQueryResult* result)
+    {
+      addResultsAsGraphics(result, m_citySymbol);
+    });
+
+    // connect to county sublayer query signal
+    connect(m_countiesTable, &ServiceFeatureTable::queryFeaturesCompleted, this, [this](QUuid, FeatureQueryResult* result)
+    {
+      addResultsAsGraphics(result, m_countySymbol);
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp]: Delete the `result` object

File: ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    });
+
+    // connect to county sublayer query signal
+    connect(m_statesTable, &ServiceFeatureTable::queryFeaturesCompleted, this, [this](QUuid, FeatureQueryResult* result)
+    {
+      addResultsAsGraphics(result, m_stateSymbol);
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp]: Delete the `result` object

File: ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      addResultsAsGraphics(result, m_citySymbol);
+    });
+
+    // connect to county sublayer query signal
+    connect(m_countiesTable, &ServiceFeatureTable::queryFeaturesCompleted, this, [this](QUuid, FeatureQueryResult* result)
+    {
+      addResultsAsGraphics(result, m_countySymbol);
+    });
+
+    // connect to county sublayer query signal
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp]: `connect to state sublayer query signal`

File: ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+  }
+}
+
+void QueryMapImageSublayer::query(const QString& whereClause)
+{
+  if (!m_citiesTable || !m_countiesTable || !m_statesTable || !m_mapView)
+    return;
+
+  // clear & delete previous graphics
+  const int graphicSize = m_selectionOverlay->graphics()->size();
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/QueryMapImageSublayer/QueryMapImageSublayer.cpp]: Probably should store `m_selectionOverlay->graphics()` in a local variable

File: ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/TokenAuthentication/TokenAuthentication.qml [hunk 5]
```diff
@@ -80,11 +80,11 @@ Rectangle {
...
     // Uncomment this section when running as standalone application
-    /*
+
     // Declare an AuthenticationView
     AuthenticationView {
         anchors.fill: parent
         authenticationManager: AuthenticationManager // set the authenticationManager property
     }
-    */
+
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/TokenAuthentication/TokenAuthentication.qml]: I think this still needs to remain commented out.

**Comment by ldanzinger** [comment]
thanks @michael-tims. updated

**Pull request merged**